tests/system/providers/google/cloud/datafusion/example_datafusion.py

```diff
@@ -42,21 +42,22 @@
 # [START howto_data_fusion_env_variables]
 SERVICE_ACCOUNT = os.environ.get("GCP_DATAFUSION_SERVICE_ACCOUNT")
 PROJECT_ID = os.environ.get("SYSTEM_TESTS_GCP_PROJECT")
+ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID")
 LOCATION = "europe-north1"
-DAG_ID = "example_data_fusion"
-INSTANCE_NAME = "test-instance"
+DAG_ID = "example_datafusion"
+INSTANCE_NAME = f"df-{ENV_ID}".replace("_", "-")
 INSTANCE = {
     "type": "BASIC",
     "displayName": INSTANCE_NAME,
     "dataprocServiceAccount": SERVICE_ACCOUNT,
 }
 
-BUCKET_NAME_1 = "test-datafusion-1"
-BUCKET_NAME_2 = "test-datafusion-2"
+BUCKET_NAME_1 = f"df-bucket1-{ENV_ID}".replace("_", "-")
+BUCKET_NAME_2 = f"df-bucket1-{ENV_ID}".replace("_", "-")
```

Comment:
WDYT if we add `DAG_ID` into objects' names along with `ENV_ID` to be completely safe from race conditions when multiple tests are executed in parallel? Current naming might bring problems in the future if somebody just adds GCS bucket creation with the same name in another test. It's possible, because nobody can check all existing tests before implementing a new one. Of course, the chance isn't high, but if it happened then it might require some notable time to troubleshoot.
This comment refers to `INSTANCE_NAME` as well and the second test too.

Reply:
For the instance name this is the only valid name, because if using also DAG_IS it exceeds the maximum amount of characters. 
As for the bucket_name, i agree on changing the name you have propose